include/ftsh_functions.h

```diff
@@ -0,0 +1,32 @@
+#ifndef FTSH_FUNCTIONS
+#define FTSH_FUNCTIONS
```

Comment:
The variable naming scheme here is not standardized across source files. Please use `_FULLTEXT_NAME_H_` across the board (that is, this would be `_FULLTEXT_FTSH_FUNCTIONS_H_`).